### PR TITLE
Remove dependency on storify from controllers/general.py

### PR DESCRIPTION
**Pull request by sl1-1**
Replaces #797 has requested changes by charmander

### CODE REVIEW

File: weasyl/controllers/general.py [hunk 1]
```diff
@@ -77,12 +82,12 @@ def search_(request):
             macro.MACRO_SUBCAT_LIST,
             search.COUNT_LIMIT,
         ]))
-    elif form.find:
-        query = search.browse(request.userid, rating, 66, form)
+    elif find:
+        query = search.browse(request.userid, rating, 66, find, cat, backid, nextid)
```
Comment (charmander) [inline weasyl/controllers/general.py:86]: Here too?

```suggestion
        query = search.browse(
            userid=request.userid,
            rating=rating,
            limit=66,
            find=find,
            cat=cat,
            backid=backid,
            nextid=nextid,
        )
```

**Review by charmander — CHANGES_REQUESTED** [review]

File: weasyl/controllers/general.py (hunk 1 above)
Comment (sl1-1) [reply weasyl/controllers/general.py]: Works for me!

**Review by charmander — APPROVED** [review]
Thanks!

**Comment by sl1-1** [comment]
Related to #744 
(just so it shows up on that one, so we can easily find all the related pull requests

**Pull request merged**